### ISSUE TITLE
Absolute paths converted to relative in declaration files

### ISSUE THREAD

**Issue by arnaudpourbaix** [ivogabe/gulp-typescript#71]
I have this reference file:
```
/// <reference path="//myway-atelier\DeclarationsTS\v1\scripts\typings\angularjs\angular.d.ts" />
/// <reference path="D:\Ressources\CDN\socle-fonctionnel.d.ts" />
```

Then, I have a task that generate js and declaration.
In my output js:
```
/// <reference path="//myway-atelier\DeclarationsTS\v1\scripts\typings\angularjs\angular.d.ts" />
/// <reference path="D:\Ressources\CDN\socle-fonctionnel.d.ts" />
```

Good, absolute paths are untouched, even if they are useless in this file.
Finally, my declaration file:
```
/// <reference path="//myway-atelier/DeclarationsTS/v1/scripts/typings/angularjs/angular.d.ts" />
/// <reference path="../../../../../../../Ressources/CDN/socle-fonctionnel.d.ts" />
```
Bad, for some reason, it is converted in relative path.
Loading this declaration file from another location will result in error because it can't find this declaration file.
As a side note, my network path is kept untouched, which is good. But I don't understand why local one is not.

In my setup, I need to keep absolute paths because all my declarations are generated into the same location.
Wouldn't it possible to have an option: "convert absolute path" ? Or maybe it is just a bug?

**Comment by arnaudpourbaix** [comment]
I have made some researchs and found this:
https://www.npmjs.com/package/gulp-tsc#path-modification
-outDir seems to be the solution for my problem, but it is not implemented in this task.
I guess that without it, both tasks have a similar behavior.

I haven't successfully done it with other task through, I still got some unrelated errors.

**Comment by ivogabe** [comment]
I'm not sure whether `outDir` will fix it, can you first try it with the `tsc` command line interface?

**Comment by arnaudpourbaix** [comment]
I have just tested and ```outdir``` fixed it.
I will get this in my declaration file:
```/// <reference path="socle-fonctionnel.d.ts" />```
Please note that it is still converted into a relative path, but at least, it leads to an existing file.
Then I tested to import this declaration file into an application and I can compile it without any error, so I assume it is working.

**Comment by ivogabe** [comment]
outDir is implemented in #97

**Comment by azarus** [comment]
Since Typescript does not perform transformations on paths the following script can be used to transform the outpaths:

https://gist.github.com/azarus/f369ee2ab0283ba0793b0ccf0e9ec590

Going to add browserify & tsify later today.
